app/src/main/java/org/mifos/mobile/ui/fragments/SavingsAccountApplicationFragment.kt

```diff
@@ -122,7 +122,11 @@ class SavingsAccountApplicationFragment : BaseFragment(), SavingsAccountApplicat
     private fun submitSavingsAccountApplication() {
         val payload = SavingsAccountApplicationPayload()
         payload.clientId = template?.clientId
+        if(productIdList.indexOf(productIdField!!.text.toString())!=-1)
         payload.productId = productIdList.indexOf(productIdField!!.text.toString()).let { productOptions?.get(it)?.id }
+        else {
+            Toast.makeText(activity,"Select a Product Id",Toast.LENGTH_SHORT).show()
```

Comment:
- Use string resources instead of hard-coded string
- Method should return immediately after showing the toast
- combine the two commits into a single one

Reply:
@rahul-gill i will use the string resource but can you help me out with the 2nd point of yours, I don't think I exactly know what you mean

Reply:
There's no use executing code after line 129 if the product id is not selected, so the function should return after showing the toast.

Reply:
@i think i got your point, also can i push the upgraded version of com.github.Raizlabs.DBFlow:dbflow-core:4.1.2. in the next commit ?

Reply:
> @i think i got your point, also can i push the upgraded version of com.github.Raizlabs.DBFlow:dbflow-core:4.1.2. in the next commit ?

@rahul-gill